tests/libtest.rs

```diff
@@ -0,0 +1,50 @@
+use std::{fs, io::Read as _};
+
+use cucumber::{given, then, when, writer, World as _, WriterExt as _};
+use regex::Regex;
+use tempfile::NamedTempFile;
+
+#[given(regex = r"(\d+) secs?")]
+#[when(regex = r"(\d+) secs?")]
+#[then(regex = r"(\d+) secs?")]
+fn step(world: &mut World) {
+    world.0 += 1;
+    assert!(world.0 < 4, "Too much!");
+}
+
+#[tokio::main]
+async fn main() {
+    let mut file = NamedTempFile::new().unwrap();
+    drop(
+        World::cucumber()
+            .with_writer(
+                writer::Libtest::new(file.reopen().unwrap()).normalized(),
```

Comment:
@ilslv let's make `::new()` return already normalized one, while a `::raw()` will return a raw `Writer`. As we do for others.